readme.md

```diff
@@ -52,7 +52,7 @@ The Chrome extension was available prior to Lighthouse being available in Chrome
 
 The Node CLI provides the most flexibility in how Lighthouse runs can be configured and reported. Users who want more advanced usage, or want to run Lighthouse in an automated fashion should use the Node CLI.
 
-_Lighthouse requires Node 14 LTS (14.x) or later._
+_Lighthouse requires Node 16 LTS (16.x) or later._
 
 **Installation**:
 
```

Comment:
Should I delete the Node 14 specific language for icu support?

Reply:
Yeah, maybe just `"To remedy this, you can upgrade to Node 14+ or manually install ICU data"` should become `"To remedy this, you can manually install ICU data"`. There are still `small-icu` builds (some node docker images use it, for instance), just by default most people shouldn't have any issue.